android/quest/build.gradle

```diff
@@ -185,6 +185,22 @@ android {
             versionCode 1
             versionName "0.0.1"
         }
+
+        mcgovern {
```

Comment:
for now you can remove this, and rename "sid" above to "anc", also rename the appId and versionNameSuffix

Reply:
or better, "bunda"